extra/modules/greenbids-real-time-data/src/main/java/org/prebid/server/hooks/modules/greenbids/real/time/data/model/predictor/ThresholdCache.java

```diff
@@ -0,0 +1,104 @@
+package org.prebid.server.hooks.modules.greenbids.real.time.data.model.predictor;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.benmanes.caffeine.cache.Cache;
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageException;
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import org.prebid.server.exception.PreBidException;
+import org.prebid.server.hooks.modules.greenbids.real.time.data.core.ThrottlingThresholds;
+import org.prebid.server.log.Logger;
+import org.prebid.server.log.LoggerFactory;
+
+import java.io.IOException;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class ThresholdCache {
+
+    private static final Logger logger = LoggerFactory.getLogger(ThresholdCache.class);
+
+    String gcsBucketName;
+
+    Cache<String, ThrottlingThresholds> cache;
+
+    Storage storage;
+
+    ObjectMapper mapper;
+
+    String thresholdsCacheKeyPrefix;
+
+    AtomicBoolean isFetching;
+
+    Vertx vertx;
```

Comment:
Please, add `private final` modifiers for all fields in all services.

Reply:
added private final in `GreenbidsInferenceDataService`, `ThresholdCache`, `ModelCache`